sample-application/devices-state-manager/DevicesStateManager/README.md

```diff
@@ -43,11 +43,33 @@ docker run devices-state-manager .
 
 ## Check results
 
-You can access DeviceStateManager health check by accessing the endpoint `http://localhost:5000/health`.
-
-When application starts running, you should see messages being received and processed
+If you run the whole solution, including the Devices Simulator, then when Devices State Manager starts running, you should see messages being received and processed
 
 ```text
 info: DevicesStateManager.EventHubReceiverService[0]
 Received event: {"deviceId": "device-42", "deviceTimestamp": "2023-07-17T17:41:52.8690130Z", "temp": 23.70572735914296}
 ```
+
+## Health check
+
+Devices State Manager has health-check implemented as a tcp probe so that kubernetes can use it in the readiness probe. If you want to check health-check locally you can execute following steps:
```

Comment:
```suggestion
Devices State Manager has health-check implemented as a TCP probe so that Kubernetes can use it in the readiness probe. If you want to check the health-check locally, you can execute following steps:
```